### PR TITLE
Remove flipper dependency for Turbo

### PR DESCRIPTION
**Pull request by rluodev**
See issue #4722

### CODE REVIEW

File: app/views/users/edit_featurepreviews.html.erb [hunk 1]
```diff
@@ -46,20 +46,6 @@
...
-                name: "Turbo Async Pages",
-                description: "Opting into this preview will enable <a href=\"https://turbo.hotwired.dev\" target=\"_blank\">Turbo</a>, a framework for speeding up web application UIs (like HCB's!) Possible side effects include:
-                  <ul>
-                    <li>Faster (sometimes instant!) page transitions</li>
-                    <li>Possibly broken UI behavior</li>
-                    <li>Flashes of cached content</li>
-                  </ul>",
-                footer: "This feature is an early tech preview and <strong>will have bugs</strong>. Please let us know when you encounter them!"
-              } %>
-
```
Comment (sampoder) [inline app/views/users/edit_featurepreviews.html.erb]: Let's keep this, it's nice to have an archive

Comment (garyhtou) [reply app/views/users/edit_featurepreviews.html.erb]: we may need to update the partial to be able to handle not passing a feature flag. Make it takes in an additional params `fully_shipped` OR `feature_flag: false`

Comment (rluodev) [reply app/views/users/edit_featurepreviews.html.erb]: Will do

File: app/views/users/_feature_preview.html.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <div class="card pb0 mb3" id="<%= local_assigns[:id] %>">
     <div class="card__banner card__banner--top flex justify-between items-center <%= local_assigns[:classes]&.any? ? local_assigns[:classes].join(" ") : "feature-generic" %>">
       <h3 class="h1 pt2 pb2 my0 color-black"><%= local_assigns[:name] %></h3>
-      <% if Flipper.enabled?(local_assigns[:feature_flag]) %>
+      <% if local_assigns[:perm_enabled] || if Flipper.enabled?(local_assigns[:feature_flag]) %>
```
Comment (sampoder) [inline app/views/users/_feature_preview.html.erb]:
```suggestion
      <% if local_assigns[:fully_shipped] || Flipper.enabled?(local_assigns[:feature_flag]) %>
```

File: app/views/users/_feature_preview.html.erb [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <div class="card pb0 mb3" id="<%= local_assigns[:id] %>">
     <div class="card__banner card__banner--top flex justify-between items-center <%= local_assigns[:classes]&.any? ? local_assigns[:classes].join(" ") : "feature-generic" %>">
       <h3 class="h1 pt2 pb2 my0 color-black"><%= local_assigns[:name] %></h3>
-      <% if Flipper.enabled?(local_assigns[:feature_flag]) %>
+      <% if local_assigns[:fully_shipped] || if Flipper.enabled?(local_assigns[:feature_flag]) %>
```
Comment (sampoder) [inline app/views/users/_feature_preview.html.erb]:
```suggestion
      <% if local_assigns[:fully_shipped] || Flipper.enabled?(local_assigns[:feature_flag]) %>
```

**Review by sampoder — APPROVED** [review]
Looks good to me! Thanks Ruien.

File: app/views/layouts/admin.html.erb [hunk 4]
```diff
@@ -16,7 +16,7 @@
     <link rel="icon" type="image/png" sizes="16x16" href="/favicon-<%= Rails.env %>-16x16.png">
     <%= yield(:head) if content_for?(:head) %>
   </head>
-  <body data-turbo="<%= Flipper.enabled?(:turbo_2023_01_23, current_user) %>">
+  <body data-turbo="true">
```
Comment (garyhtou) [inline app/views/layouts/admin.html.erb]: This should work just fine, but I wonder if the `data-turbo="true"` is unnecessary. The docs aren't super clear; maybe we can find out via testing

cc @cjdenio

File: app/views/users/_feature_preview.html.erb [hunk 5]
```diff
@@ -25,3 +25,4 @@
     <% end %>
   </div>
 <% end %>
+<% end %>
```
Comment (garyhtou) [inline app/views/users/_feature_preview.html.erb]: is this an extra `<% end %>`?

File: app/views/users/edit_featurepreviews.html.erb [hunk 6]
```diff
@@ -49,7 +49,8 @@
           <%= render partial: "users/feature_preview", locals: {
                 id: "turbo",
                 classes: ["feature-turbo"],
-                feature_flag: :turbo_2023_01_23,
+                feature_flag: :none,
```
Comment (garyhtou) [inline app/views/users/edit_featurepreviews.html.erb]: We should to remove that entirely
```suggestion
```

**Review by garyhtou — DISMISSED** [review]
couple small changes

**Review by cjdenio — APPROVED** [review]

**Pull request merged**